Changelog.md

```diff
@@ -4,6 +4,7 @@ __New Features__
 
 __Bugfixes__
 - Allows additional flexibility for multiple `Attic`/`Foundation` elements.
+- ReportUtilityBills measure: apply production compensation defaults to batteries and generators.
```

Comment:
Should we update the "PV Compensation" section of the docs? E.g., replace all "PV" with "Production"?